src/Data/List/Any/Properties/CommutativeMonoid.agda

```diff
@@ -0,0 +1,136 @@
+open import Algebra using (CommutativeMonoid)
+
+module Data.List.Any.Properties.CommutativeMonoid {c ℓ} (commutativeMonoid : CommutativeMonoid c ℓ) where
+
+open import Data.Nat using (ℕ)
+open import Data.Fin as F using (Fin)
+import Data.Fin.Properties as FP
+open import Data.Product using (∃; _,_; proj₁; proj₂)
+open import Function using (_∘_)
+open import Data.List hiding (sum)
+open import Data.List.Any
+open import Data.List.Any.Membership.Propositional
+open import Data.Fin.Vec
+open import Data.Maybe using (Maybe; just; nothing; maybe)
+
+open import Relation.Binary using (Setoid)
+import Relation.Binary.EqReasoning as EqReasoning
+import Relation.Binary.Sigma.Propositional as OverΣ
+open OverΣ using (OverΣ)
+open import Relation.Binary.PropositionalEquality as P using (_≡_; _≗_)
+import Relation.Binary.Indexed as I
+import Function.Related as FR
+open import Function.Inverse renaming (_∘_ to _∘ⁱ_; sym to symⁱ; id to idⁱ)
+import Function.Equality as FunEq
+open FunEq using (_⟶_; _⟨$⟩_)
+
+open CommutativeMonoid commutativeMonoid renaming (_∙_ to _+_)
+open import Algebra.Operations.CommutativeMonoid commutativeMonoid
+open import Algebra.Properties.CommutativeMonoid commutativeMonoid
+
+module B = Setoid ([ bag ]-Equality Carrier)
+
+private
```

Comment:
No need for this to be private

Reply:
I made it private because it doesn't really make sense for those definitions to be exported from this module. I agree that they'd be useful elsewhere though!